Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerChangedGameStatusMessage.java

```diff
@@ -6,17 +6,29 @@
 import nl.soccar.socnet.message.MessageEvent;
 
 /**
+ * Message for changing the gamestatus of the game.
+ * 
  * @author PTS34A
  */
 @MessageEvent(id = MessageConstants.PLAYER_CHANGED_GAME_STATUS_MESSAGE_ID, handler = PlayerChangedGameStatusMessageHandler.class)
 public final class PlayerChangedGameStatusMessage extends Message {
 
     private final Status status;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)